src/index.html

```diff
@@ -6,7 +6,69 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <section class="page__calendar calendar calendar--start-day--sun calendar--month-length--28">
```

Comment:
Не впевнений що це конкретно помилка, але я думаю що в БЕМ на першому місці повинен стояти клас Блоку, якщо це блок, а потім класи елементів.
Думаю так легше одразу відрізняти блоки в структурі і не пропустити коли він захований між іншими тегами
```suggestion
    <section class="calendar page__calendar  calendar--start-day--sun calendar--month-length--28">
```